docs/doc/00-overview/index.md

```diff
@@ -18,7 +18,7 @@ import TabItem from '@theme/TabItem';
 <Tabs groupId="whydatabend">
 <TabItem value="Performance" label="Performance">
 
-- Blazing-fast data analytics on object storage.
+- [Blazing-fast data analytics](https://benchmark.clickhouse.com/) on object storage.
```

Comment:
[Proved performance by ClickBench](https://benchmark.clickhouse.com/) for data analytics on object storage.

Reply:
That's not proven, clickbench is not benched on object storage, I will update it in another.